libraries/AP_DDS/README.md

```diff
@@ -67,7 +72,7 @@ section, you should remove it and switch to local install.
 
 | Name | Description |
 | - | - |
-| SERIAL1_BAUD | The serial baud rate for DDS |
+| SERIAL1_BAUD | The serial baud rate for DDS, use 115 |
```

Comment:
I found this weird, we should recommend higher baudrate by default I think. @Ryanf55 any issue on microros to raise this to 1MB ?

Reply:
1MB serial is too fast for noisy drones, I'd rather just let the user decide what to use for their application. On SITL, since it's USB, the baud has no effect, at least from what Tridge told me.

Reply:
indeed, it doesn't affect USB, but for UARTs it is common to use 921600, and it doesn't have any problems with noise even with quite long cables